github/snapcraft.py

```diff
@@ -1,37 +1,126 @@
-#!/usr/bin/env python2
+#!/usr/bin/env/python2
```

Comment:
This is a bogus change. You shouldn't modify it, so please change the / back to a space.

Reply:
oh, wait, actually this python files has no main, so this shebang is not doing anything here.
You can just remove the line.